sdk/ml/azure-ai-ml/azure/ai/ml/operations/_data_operations.py

```diff
@@ -566,7 +566,16 @@ def _get_latest_version(self, name: str) -> Data:
 
     @monitor_with_activity(logger, "data.Share", ActivityType.PUBLICAPI)
     @experimental
-    def share(self, name, version, *, share_with_name, share_with_version, registry_name) -> Data:
+    def share(
+        self,
+        name,
+        version,
+        *,
+        share_with_name,
+        share_with_version,
+        registry_name,
+        **kwargs,  # pylint:disable=unused-argument
```

Comment:
pass the kwargs to the restclient operation

Reply:
create_or_update uses scope_kwargs and init_kwargs. So just passed kwargs to get operation